config/application.rb

```diff
@@ -30,5 +30,7 @@ class Application < Rails::Application
     #
     # config.time_zone = "Central Time (US & Canada)"
     # config.eager_load_paths << Rails.root.join("extras")
+
+    config.assets.prefix = "/assets/collections-publisher"
```

Comment:
Copy-pasto?

Reply:
Its a good job one of us is paying attention. Fixed